config.go

```diff
@@ -61,7 +61,11 @@ func newDefaultConfig() *Config {
 
 // WithOAuthURI returns the oauth uri
 func (r *Config) WithOAuthURI(uri string) string {
-	return fmt.Sprintf("%s/%s", r.OAuthURI, uri)
+	if r.BaseURI != "" {
```

Comment:
perhaps we can use the below rather than if statement

```Go
filepath.Join(r.BaseURI, r.OAuthURI, uri)
```

as it will handle a zero string anyhow ..

Reply:
I had it in mind while typing, but somehow felt a mismatch between URI and **file** path (read: backslashes on Windows) - yet I don't expect anyone to run this proxy on Windows, so I think it's fine.

Reply:
make's sense ... we should keep the `if` for now ..